css/index.css

```diff
@@ -1,3 +1,269 @@
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+	body {
+		max-width: 100%;
+		margin: 0 auto;
+		
+	}
+
+	header {
+        display: flex;
+        justify-content: space-between;
+		align-items: center;
+		border: teal;
+	}
+
+	header nav {
+        width: 100%;
+        display: flex;
+        justify-content: space-evenly;
+        align-items: center;
+		margin: auto 0px 0px;
+		padding: 10px;
+    }
+    
+    header nav a {
+		display: flex;
+        color: gray;
+        text-decoration: none;
+		padding: 10px;
+		font-weight: bold;
+		font-size: 15px;
+		justify-content: space-evenly;
+	}
+	
+	header .logo {
+		padding: 20px;
+	}
+
+
+
+	.top-sec {
+		display: flex;
+		justify-content: space-around;
+		padding: 20px;
+		border: 2px solid teal;
+	}
+
+	.top-sec .cta {
+		align-content: center;
+		text-align: center;
+		display: flex;
+		flex-direction: column;
+		
+	}
+
+	.top-sec .cta h1 {
+		font-size: 4rem;
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+		justify-content: space-around;
+		padding: 10px;
+	}
+
+	.top-sec .cta h1 button {
+		background-color: white;
+		color: black;
+		text-align: center;
+		font-size: 13px;
+		font-weight: bold;
+		border: 1px solid black;
+		width: 50%;
+	}
+
+
+	.mid-top {
+		display: flex;
+        border-top: 2px solid black;
+		padding: 30px;
+	
+	}
+
+	.mid-top .mid-right {
+		display: flex;
+		flex-direction: column;
+		padding: 20px;
+		align-content: center;
+	}
+
+	.mid-top .mid-left {
+		display: flex;
+		flex-direction: column;
+		padding: 20px;
+		align-content: center;
+	}
+
+	.mid-top .middle-img {
+		display: inline-block;
+		border: 4px dotted rosybrown;
+		padding: 20px;
+	}
```

Comment:
```suggestion
max-width: 100%;
	}
```

Reply:
adding this will constrain the image to its parent container and cause the image to scale down when screen size is less